src/Fuse/Fuse.csproj

```diff
@@ -1,188 +1,29 @@
-    <Compile Include="compute\DynamicStructGetAttribute.cs" />
-    <Compile Include="compute\DynamicStructBufferGet.cs" />
-    <Compile Include="compute\TypedBuffer.cs" />
-    <Compile Include="compute\DynamicStruct.cs" />
-    <Compile Include="compute\DynamicStructSetAttribute.cs" />
-    <Compile Include="compute\DynamicStructBufferSet.cs" />
-    <Compile Include="compute\GpuVoid.cs" />
-    <Compile Include="ConstantValue.cs" />
-    <Compile Include="DelegateNode.cs" />
-    <Compile Include="DrawShader.cs" />
-    <Compile Include="Float2Join.cs" />
-    <Compile Include="Float3Join.cs" />
-    <Compile Include="Float4Join.cs" />
-    <Compile Include="FromFloat.cs" />
-    <Compile Include="FunctionNode.cs" />
-    <Compile Include="GPUInput.cs" />
-    <Compile Include="GpuStruct.cs" />
-    <Compile Include="GpuStructNode.cs" />
-    <Compile Include="GpuValue.cs" />
-    <Compile Include="GetMember.cs" />
-    <Compile Include="noise\NoiseInflection.cs" />
-    <Compile Include="noise\Noisetype.cs" />
-    <Compile Include="noise\WorleyCellFunction.cs" />
-    <Compile Include="noise\WorleyCellDistance.cs" />
-    <Compile Include="NumericSwitchNode..cs" />
-    <Compile Include="OperatorNode.cs" />
-    <Compile Include="DeclareValue.cs" />
-    <Compile Include="particles\ParticleAttribute.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="regions\GpuFunctionRegion.cs" />
-    <Compile Include="regions\GpuFunctionRegionEmit.cs" />
-    <Compile Include="SampleTextureNode.cs" />
-    <Compile Include="sdf\CombineSDFSelector.cs" />
-    <Compile Include="sdf\DistanceMetricsSelector.cs" />
-    <Compile Include="sdf\Ray.cs" />
-    <Compile Include="Semantic.cs" />
-    <Compile Include="SetMember.cs" />
-    <Compile Include="ShaderNode.cs" />
-    <Compile Include="ShaderNodesUtil.cs" />
-    <Compile Include="BooleanSwitchNode.cs" />
-    <Compile Include="Tests\TestDynamicStruct.cs" />
-    <Compile Include="Tests\TestControl.cs" />
-    <Compile Include="Tests\TestDelegate.cs" />
-    <Compile Include="Tests\TestFunction.cs" />
-    <Compile Include="Tests\TestGraph.cs" />
-    <Compile Include="ToFloat4.cs" />
-    <Compile Include="AbstractToShaderFX.cs" />
-    <Compile Include="TypeHelpers.cs" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-         Other similar extension points exist, see Microsoft.Common.targets.
-    <Target Name="BeforeBuild">
-    </Target>
-    <Target Name="AfterBuild">
-    </Target>
-    -->
-</Project>
\ No newline at end of file
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net48</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Stride" Version="4.0.1.1401" />

```

Comment:
nugets are now transitive, you only need the top nuget, like Engine, which references things like core and so on... so maybe start with Engine and then see what's missing.

Reply:
ah, great. only stride.engine is enough, see my last commit